packages/forms/resources/views/components/tabs.blade.php

```diff
@@ -55,19 +55,31 @@
                 role="tab"
                 x-bind:tabindex="tab === '{{ $tab->getId() }}' ? 0 : -1"
                 class="flex items-center gap-2 shrink-0 p-3 text-sm font-medium"
-                x-bind:class="{ 'bg-white @if (config('forms.dark_mode')) dark:bg-gray-800 @endif': tab === '{{ $tab->getId() }}' }"
+                x-bind:class="{
+                    'text-gray-400 dark:text-gray-400': tab !== '{{ $tab->getId() }}',
+                    'bg-white text-primary-600 dark:bg-gray-800': tab === '{{ $tab->getId() }}'
+                }"
             >
                 @if ($icon = $tab->getIcon())
                     <x-dynamic-component
                         :component="$icon"
-                        :class="\Illuminate\Support\Arr::toCssClasses([
-                            'h-4 w-4 text-gray-500',
-                            'dark:text-gray-400' => config('forms.dark_mode'),
-                        ])"
+                        class="h-5 w-5"
                     />
                 @endif
 
-                {{ $tab->getLabel() }}
+                <span>{{ $tab->getLabel() }}</span>
+
+                @if ($tab->getBadge())
+                    <span
+                        class="inline-flex items-center justify-center ml-auto rtl:ml-0 rtl:mr-auto min-h-4 px-2 py-0.5 text-xs font-medium tracking-tight rounded-xl whitespace-normal transition"
+                        x-bind:class="{
+                            'text-gray-400 bg-gray-500/10 dark:bg-gray-600 dark:text-gray-400': tab !== '{{ $tab->getId() }}',
```

Comment:
Isn't it better to use one of the shades of gray from the palette instead of tweaking with opacity?

Reply:
sometime the default shades doesn't provide a better contrast... specially in this situation, and i have seen it being used in filament in those cases... so just following what's already there...
we can't achive the following without using the opacity modifier
<img width="601" alt="Screen Shot 2022-07-30 at 2 53 49 AM" src="https://user-images.githubusercontent.com/10007504/181853202-d5df555d-46ca-4e8e-9083-76856001b5f5.png">

<img width="601" alt="Screen Shot 2022-07-30 at 2 53 39 AM" src="https://user-images.githubusercontent.com/10007504/181853230-ea27a7d5-9144-4c6a-a7c1-531ec8266e79.png">

or i might be wrong, never quiet get the hang of it tbh...

Reply:
sure thing!